DESCRIPTION

```diff
@@ -3,76 +3,82 @@ Type: Package
 Title: Indices of Effect Size and Standardized Parameters
 Version: 0.4.0.001
 Authors@R: c(
-  person("Mattan S.", 
-		"Ben-Shachar", 
-		role = c("aut", "cre"), 
-		email = "matanshm@post.bgu.ac.il", 
-		comment = c(ORCID = "0000-0002-4287-4801")),
-	person("Dominique", 
-		"Makowski", 
-		email = "dom.makowski@gmail.com",
-		role = c("aut"), 
-		comment = c(ORCID = "0000-0001-5375-9967")),
-	person("Daniel", 
-		"Lüdecke", 
-		role = c("aut"), 
-		email = "d.luedecke@uke.de", 
-		comment = c(ORCID = "0000-0002-8895-3206")),
-	person("Ken", 
-		"Kelley", 
-		role = c("ctb")),
-	person("David", 
-		"Stanley", 
-		role = c("ctb"))
-	)
+    person("Mattan S.", 
+        "Ben-Shachar", 
+        role = c("aut", "cre"), 
+        email = "matanshm@post.bgu.ac.il", 
+        comment = c(ORCID = "0000-0002-4287-4801")),
+    person("Dominique", 
+        "Makowski", 
+        role = c("aut"),
+        email = "dom.makowski@gmail.com",
+        comment = c(ORCID = "0000-0001-5375-9967")),
+    person("Daniel", 
+        "Lüdecke", 
+        role = c("aut"), 
+        email = "d.luedecke@uke.de", 
+        comment = c(ORCID = "0000-0002-8895-3206")),
+    person("Indrajeet", 
+        "Patil", 
+        role = c("ctb"), 
+        email = "patilindrajeet.science@gmail.com", 
+        comment = c(ORCID = "0000-0003-1995-6531", Twitter = "@patilindrajeets")),
```

Comment:
You can add twitter???? :O

Reply:
Haha, yes!

Not only that, `pkgdown` will also render it properly

<img width="238" alt="Screenshot 2020-12-02 at 09 37 34" src="https://user-images.githubusercontent.com/11330453/100848905-1523e980-3482-11eb-8bc2-3cb95c8d6356.png">